main.go

```diff
@@ -732,9 +734,37 @@ func executeLinter(state *linterState, args []string) error {
 			state.Name)
 	}
 
-	processOutput(state, buf.Bytes())
+	cmdOutput, err := preprocessLinterPanic(state.Name, buf, err)
+	processOutput(state, cmdOutput)
 	debug("%s linter took %s", state.Name, time.Since(start))
-	return nil
+	return err
+}
+
+const panicExitStatus = 2
+
+func preprocessLinterPanic(name string, buf *bytes.Buffer, processError error) ([]byte, error) {
+	if exitStatus(processError) == panicExitStatus {
+		var err error
+		cmdOutput := new(bytes.Buffer)
+		routines, _ := stack.ParseDump(buf, cmdOutput)
```

Comment:
`stack.ParseDump` [strips all output](https://godoc.org/github.com/maruel/panicparse/stack#ParseDump) it doesn't understand, and this functions returns that output. Perhaps it'd just be better to use ParseDump to check routines, but discard the output? I.e. `routines, _ := stack.ParseDump(buf, ioutil.Discard)`.

I say this because this function can never reliably say whether the output is a panic (different Go versions may alter the panic output and exit status), so perhaps a more conservative approach may be best?